src/main/webapp/WEB-INF/tags/instructor/feedbackEdit/newQuestionForm.tag

```diff
@@ -119,8 +119,10 @@
           
             <div>
                 <span class="pull-right">
-                    <input id="button_submit_add" class="btn btn-primary"
-                        type="submit" value="Save Question" tabindex="9">
+                    <button id="button_submit_add" class="btn btn-primary"
+                        type="submit" tabindex="9">
```

Comment:
Both should be fine on the previous line, even though it was line-broken before.